stdlib/warnings.pyi

```diff
@@ -2,7 +2,7 @@ from types import ModuleType, TracebackType
 from typing import Any, List, TextIO, Type, overload
 from typing_extensions import Literal
 
-from _warnings import warn as warn, warn_explicit as warn_explicit
+from _warnings import filters as filters, warn as warn, warn_explicit as warn_explicit
```

Comment:
Mutating `filters` directly, e.g. `filters.append(foo)`, is apparently a bad idea. There's a function named `_filters_mutated()` in `_warnings.py` that should get called, but wouldn't. I think it would be better to use something like `filters: Sequence[Tuple[Any, ...]]` to prevent accidental mutating.

While you're at it, you could also improve the type of `filters`. Currently It is defined as `List[Tuple[Any, ...]]`, but we could be more specific than "list of tuples".

```python
>>> from warnings import filters
>>> filters
[('default', None, <class 'DeprecationWarning'>, '__main__', 0), ('ignore', None, <class 'DeprecationWarning'>, None, 0), ('ignore', None, <class 'PendingDeprecationWarning'>, None, 0), ('ignore', None, <class 'ImportWarning'>, None, 0), ('ignore', None, <class 'ResourceWarning'>, None, 0)]
>>> 
```